packaging/Docker/Dockerfile.dev-minimal

```diff
@@ -22,8 +22,8 @@ RUN dnf --refresh -y install \
 	spdlog-devel
 
 # Python dependencies
-ADD requirements.txt .
+ADD project.toml .
 RUN pip3 install --upgrade wheel build
-RUN pip3 install -r requirements.txt
+RUN pip3 install .[test,docs]
 
 EXPOSE 8888
```

Comment:
Why do we expose 8888 here? LIkely left from some jupyter

Reply:
This is intended to be able to start and use jupyterlab from within the dev container.

Reply:
I think its not required. `EXPOSE` directives are not mandatory for accessing the container. Its only used by an "expose all ports" setting. 

I will remove it here as we dont even install Jupyter in this Dockerfile.
It might make sense in the Dockerfile in which. Jupyter is actually installed